gradle/dependencies.gradle

```diff
@@ -21,6 +21,10 @@ ext.aosp_libs = fileTree(include: [
     'android.jar',
     // out/target/common/obj/JAVA_LIBRARIES/core-libart.com.android.art_intermediates/classes.jar
     'libcore.jar',
+    // out/target/product/device/obj/JAVA_LIBRARIES/setupcompat_intermediates/javalib.jar
+    'setupcompat.jar',
+    // out/target/product/device/obj/JAVA_LIBRARIES/setupdesign_intermediates/javalib.jar
+    'setupdesign.jar',
```

Comment:
If those are not automatically generated, we should also document on how to get them.

Reply:
I wrote it in the commit message I think, it's a simple:

`m setupcompat setupdesign`